test/cpp/strings.unit.cpp

```diff
@@ -6,123 +6,114 @@
 #include <microstrain/strings.hpp>
 
 static constexpr char CHECK_STRING[] = "Test: \"quotes\" 'single' & <xml/> {json} [array] $100 €50 ¥200 @user #tag 🚀 \n\t\\ 100% café naïve";
-size_t CHECK_STRING_LENGTH = 102; // Length without null terminator
-
-struct BufferWrapper
-{
-    char array[1024] = {};
-    microstrain::Span<char> span{array};
-    size_t index = 0;
-};
+static constexpr size_t CHECK_STRING_LENGTH = 102; // Length without null terminator
 
 // TODO: Figure out naming for these tests
 
-TEST("Span string concatenation", "A C string can be concatenated to an explicitly created span")
-{
-    BufferWrapper buffer{};
-
-    const bool ok = microstrain::strings::concat(buffer.span, &buffer.index, CHECK_STRING, CHECK_STRING_LENGTH);
-
-    EXPECT_TO_BE_TRUE(ok);
-    EXPECT_C_STRINGS_TO_BE_EQUAL(CHECK_STRING, buffer.array);
-    EXPECT_BUFFER_TO_BE_TERMINATED_AT_POSITION(CHECK_STRING_LENGTH, buffer.array);
-}
-
-TEST("Span string concatenation", "A C string can be concatenated to an implicitly created span")
+TEST("Span string concatenation", "A C string can be concatenated to a span")
```

Comment:
It's not really concatenating to the span, it's concatenating to the string contained in the buffer. The span here is incidental, and just the way the buffer is passed in. Since that's the only API provided, there's no need to call it to attention. How about "A C string can be appended"?